src/core/operation.c

```diff
@@ -179,6 +183,7 @@ QuicOperationDequeue(
 #if DEBUG
         Oper->Link.Flink = NULL;
 #endif
+        QuicPerfCounterDecrement(QUIC_PERF_COUNTER_OPER_QUEUE_DEPTH);
```

Comment:
Maybe put this outside the lock, only if `Oper` isn't null.

Reply:
The conundrum was whether adding another if-statement hurt performance more than doing the interlocked operation under lock.  The if-statement is probably faster by far.

Reply:
Yeah, the if will be faster, especially once trained with PGO.